Source/SPUD/Private/SpudState.cpp

```diff
@@ -28,7 +28,10 @@ void USpudState::ResetState()
 
 void USpudState::StoreWorldGlobals(UWorld* World)
 {
-	SaveData.GlobalData.CurrentLevel = World->GetFName().ToString();
+	if (UPackage* Package = World->GetPackage())
+	{
+	    SaveData.GlobalData.CurrentLevel = Package->GetLoadedPath().GetPackageFName().ToString();
+	}
 }
```

Comment:
I think perhaps a fallback to the old behaviour as an "else" would make sense here, just in case.

Reply:
Yeah, probably you're right